File: ballast-core/dotnet/src/application/models/CreateGameOptions.cs
namespace Ballast.Core.Application.Models
{
    public class CreateGameOptions
    {
        public CreateVesselOptions[] VesselOptions { get; set; }
        public string BoardType { get; set; }
        public int? BoardSize { get; set; }
        public string BoardShape { get; set; }
        public double? LandToWaterRatio { get; set; }
    }
}

File: ballast-core/npm/src/application/models/create-game-options.ts
import { ICreateVesselOptions } from "./create-vessel-options";

export interface ICreateGameOptions {
    vesselOptions: ICreateVesselOptions[];
    boardSize: number | null;
    boardType: string | null;
    boardShape: string | null;
    landToWaterRatio: number | null;
}